### DISCUSSION TITLE
How to add Images in the Others page ??

### DISCUSSION THREAD

**Discussion by tushaarkataria** [academicpages/academicpages.github.io#1990] [Q&A] [answered]
We can add/embed images in blog post. But we can't do in other pages like "Others".

I just want to be able to add an image and show it displayed in the 'others' page.

I tried adding images a markdown by placing them in the same folder, in images folder and then giving a full path and <img src=""> html code. But none of it worked. Shows up as the image shown below for some reason.

![Screenshot_20240314_133916](https://github.com/academicpages/academicpages.github.io/assets/77859502/590f9fa8-be31-4305-9afa-224b1afe94b7)

**Comment by rjzupkoii** [comment]
I'm not sure what you mean by the full path, but you supply the path relative to the root of the repository (i.e., `<img src='/images/500x300.png'>`) then they should render correctly. There are some examples of this in the [_portfolio](https://github.com/academicpages/academicpages.github.io/tree/master/_portfolio) directory of the template.

**Comment by tushaarkataria** [comment]
I am not using portfolio page, but i thought that it shouldn't have mattered. 

I have tried multiple things for embedding images in the webpage.

`<img src='https://github.com/tushaarkataria/tushaarkataria.github.io/blob/master/images/IMG_1153.png'>`
And 
`<img src='IMG_1153.png'>`
And
`<img src='IMG_1153.JPG'>`

The image shows up well in the md page.
https://github.com/tushaarkataria/tushaarkataria.github.io/blob/master/_pages/other.md

But not on the website.
https://tushaarkataria.github.io/other/

**Reply by rjzupkoii** [reply, marked as answer]
I appreciate the pathing can be a bit confusing, but the reason I mentioned the [_portfolio](https://github.com/academicpages/academicpages.github.io/tree/master/_portfolio) directory because those are examples of how images need to be referenced. If you add the images to the `/images/` directory of you repository then the correct path for the image will be (note the `/images/` directory is included):

```html
<img src='/images/500x300.png'>
```

You can see that I also do this with my own website, [such as in this example](https://github.com/rjzupkoii/rjzupkoii.github.io/blob/master/_portfolio/2023-03-01-national-parks.md). The reason you need to do this is to ensure that images are served relative to your website (i.e., https://tushaarkataria.github.io) not relative to https://github.com/tushaarkataria/tushaarkataria.github.io which is a form of hot-linking. Indeed, if you try to go to https://github.com/tushaarkataria/tushaarkataria.github.io/blob/master/images/IMG_1153.png you will find that you are actually directed to the repository browser, and not served just the image as you might expect.

**Comment by Yonggie** [comment]
Can't we use markdown to show the picture at blog?

I did but it shows nothing on the rendered blog...
![图片](https://github.com/user-attachments/assets/2bd3b239-8adb-447c-9f3b-056fd5b01bc7)

![图片](https://github.com/user-attachments/assets/cfb40f56-c4f9-4134-8aac-9e7d0e9794c8)

**Reply by rjzupkoii** [reply]
Markdown for images is supported; however, the same issues with pathing to the images applies to Markdown as well.
